ehrapy/api/_anndata_util.py

```diff
@@ -36,3 +37,41 @@ def get_column_values(adata: AnnData, indices: Union[int, List[int]]) -> np.ndar
         :class:`~numpy.ndarray` object containing the column values
     """
     return np.take(adata.X, indices, axis=1)
+
+
+def get_numeric_vars(adata: AnnData) -> List[str]:
+    """Fetches the column names for numeric variables in X
+
+    Args:
+        adata: :class:`~anndata.AnnData` object
+
+    Returns:
+        Set of column numeric column names
+    """
+
+    numeric_vars: List[str] = []
+
+    return numeric_vars
+
+
+def set_numeric_vars(
+    adata: AnnData, values: Union[np.ndarray, spmatrix], vars: Optional[List[str]] = None, copy: bool = False
+) -> Optional[AnnData]:
+    """Fetches the column names for numeric variables in X
+
+    Args:
+        adata: :class:`~anndata.AnnData` object
+        values: Matrix containing the replacement values
+        vars: List of names of the numeric variables to replace. If `None` they will be detected using ~ehrapy.pp.get_numeric_vars.
+        copy: Whether to return a copy with the normalized data.
+
+    Returns:
+        :class:`~anndata.AnnData` object with updated X
+    """
+
+    adata_to_set = adata
```

Comment:
This was my ugly way of handling the copy. Think I already simplified this somewhere

```
if copy:
    adata = adata.copy()
```

should suffice I guess.